src/sentry/runner/settings.py

```diff
@@ -92,9 +92,9 @@ def configure(ctx, py, yaml, skip_service_validation=False):
     # Make sure that our warnings are always displayed.
     warnings.filterwarnings("default", "", Warning, r"^sentry")
 
-    from django.utils.deprecation import RemovedInDjango30Warning
+    from django.utils.deprecation import RemovedInDjango40Warning
```

Comment:
Would it be a good idea to import `RemovedInNextVersionWarning` here to future proof things a bit more?